src/components/Elements.css

```diff
@@ -0,0 +1,5 @@
+.btn {
+background-color: green;
```

Comment:
Please indent these lines. In CSS scope inside curly braces should be indented by 4 spaces.